modules/learn/pages/services-and-indexes/indexes/storage-modes.adoc

```diff
@@ -0,0 +1,63 @@
+= Storage Settings
+:description: All indexes in Couchbase Capella use the Couchbase Plasma storage engine. 
+:page-topic-type: concept
+:page-aliases: indexes:storage-modes,understanding-couchbase:services-and-indexes/indexes/storage-modes,architecture:index-storage
+
+[abstract]
+{description}
+
+
+[#standard-index-storage]
+== Standard Index Storage
+
+In Couchbase Capella, _Standard_ is the storage-setting for Secondary Indexes: the indexes are saved on disk; in a disk-optimized format that uses both memory and disk for index-update and scanning.
```

Comment:
Removing mention of "standard" storage, but leaving the anchor alone for cross-branch compatibility.

```suggestion
== Index Storage

In Couchbase Capella, indexes are saved on disk, in a disk-optimized format that uses both memory and disk for index-update and scanning.
```

Reply:
Yeah, I wasn't sure about mentioning "standard" storage at all. Thanks!